homework6/src/main/java/Car.java

```diff
@@ -0,0 +1,66 @@
+import java.util.Optional;
+
+public class Car {
+    private String make;
+    private String model;
+    private Flue flue;
```

Comment:
it is definitely not the FLUE it's FUEL ? :))

Reply:
da:)) am scris gresit